### PR TITLE
Tests for content script for OU selection

### PR DESCRIPTION
**Pull request by varunsivashankar**
**contentTest.html**
- This is the html file to test the content script.
- It contains a mock table for testing purposes to mimic the DOM of the OU list page.

**contentTest.js**
- This includes tests for the _addButtons_ function that will be included in the content script.
- Removed the mock table that was created for testing purposes.

### CODE REVIEW

**Comment by areeb1992** [comment]
This PR should be raised along with content script, or after it.

File: STEP_Project/tests/contentTest.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	
+    <title>Testing with Jasmine</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jasmine/2.8.0/jasmine.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jasmine/2.8.0/jasmine.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jasmine/2.8.0/jasmine-html.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jasmine/2.8.0/boot.min.js"></script>
+    <script src="../content.js"></script>
+    <script src="contentTest.js"></script>
+
+	<table id = 'table' role = 'grid'>
```
Comment (areeb1992) [inline STEP_Project/tests/contentTest.html:13]: Indentation looks wrong.

File: STEP_Project/tests/contentTest.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	
+    <title>Testing with Jasmine</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jasmine/2.8.0/jasmine.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jasmine/2.8.0/jasmine.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jasmine/2.8.0/jasmine-html.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jasmine/2.8.0/boot.min.js"></script>
+    <script src="../content.js"></script>
```
Comment (areeb1992) [inline STEP_Project/tests/contentTest.html:10]: Rename to ouselect_content.js.

Otherwise there will be name conflicts when we do it for user and group

File: STEP_Project/tests/contentTest.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+        row = OUs[1];
+        temp = row.getElementsByClassName('bClass');
+        expect(temp.length).toEqual(1); 
+
+        row = OUs[2];
+        temp = row.getElementsByClassName('bClass');
+        expect(temp.length).toEqual(1); 
+
+        tabl.remove();
```
Comment (atishey-jain) [inline STEP_Project/tests/contentTest.js:72]: Just curious, why is this needed in just this test ?

File: STEP_Project/tests/contentTest.html (hunk 1 above)
Comment (varunsivashankar) [reply STEP_Project/tests/contentTest.html]: Fixed the tabs/spaces indentation issue.

File: STEP_Project/tests/contentTest.html (hunk 2 above)
Comment (varunsivashankar) [reply STEP_Project/tests/contentTest.html]: Renamed it to OUListContent.js because it was more consistent with our naming for other files

File: STEP_Project/tests/contentTest.js (hunk 3 above)
Comment (varunsivashankar) [reply STEP_Project/tests/contentTest.js]: _tabl_ was created in the html file just for testing purposes, so I removed it after the last test. If I don't, then the table appears on the page with the results of all the tests.
